### PR TITLE
Added splat to winston logger config to enable string interpolation

### PR DESCRIPTION
**Pull request by ndonnan**
This enables string interpolation in log messages.

See [Issue #843](https://github.com/mojaloop/project/issues/843) for more details

### CODE REVIEW

**Comment by ndonnan** [comment]
> Looks good to me.
> 
> Although I'm a little curious why we need this when as I find native JS string templating to be far easier to grok:
> 
> e.g.
> 
> ```
> Logger.info(`getById::id(${id})`)
> ```
> 
> vs.
> 
> ```
> Logger.info('getById::id(%s)', id)
> ```

Yip I see that is the way it is done in the central-ledger code and is easier to read.

I found a lot of places in the ml-api-adapter that are logging using the %s method that aren't currently logging correctly due to this.

**Comment by lewisdaly** [comment]
>
> Yip I see that is the way it is done in the central-ledger code and is easier to read.
> 
> I found a lot of places in the ml-api-adapter that are logging using the %s method that aren't currently logging correctly due to this.

Cool. Just wanted to double check.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "6.4.0",
+  "version": "6.4.1",
```
Comment (mdebarros) [inline package-lock.json]: Also ensure that package.json reflects this change.

**Comment by ndonnan** [comment]
Version has been bumped to 7.0.0

**Pull request merged**